sdk/src/asset_handlers/pdf.rs

```diff
@@ -560,4 +681,42 @@ mod tests {
         let pdf = Pdf::from_bytes(include_bytes!("../../tests/fixtures/basic.pdf")).unwrap();
         assert!(pdf.read_xmp().is_some());
     }
+
+    #[cfg_attr(not(target_arch = "wasm32"), test)]
+    #[cfg_attr(target_arch = "wasm32", wasm_bindgen_test)]
+    fn test_remove_manifest_bytes_from_file_without_c2pa_returns_error() {
+        let mut pdf = Pdf::from_bytes(include_bytes!("../../tests/fixtures/basic.pdf")).unwrap();
+
+        assert!(matches!(
+            pdf.remove_manifest_bytes(),
+            Err(Error::NoManifest)
+        ));
+    }
```

Comment:
Not a blocker, but perhaps verify that `pdf` wasn't changed in any way despite the error result?

Reply:
Will take a look at this soon. Looking into a way to do this test with the embedded manifest work coming up!